### ISSUE TITLE
Add a warning for extraneous spaces in PAE

### ISSUE THREAD

**Issue by lpugin** [rism-digital/verovio#2854]
Currently, PAE code like this `'' 4 C D E F` parses without warning. However, extraneous spaces should yield a warning.

**Comment by ahankinson** [comment]
Not a blocker, but we use Verovio for parsing input queries on RISM Online, and in this case we probably do want to be fairly liberal with what we accept as a query.

Would there be any value in an option for Verovio to disable specific error/warning codes: `--pae-ignore E123,E456,E789`?

**Comment by lpugin** [comment]
Yes, this is right. Are the logs enabled in RISM Online? We probably already have quite a few warnings when using existing incipits are a query.

Being able to disable some codes would be useful.
